IFC 4.3/UT_Alignment_12d_1/UT_Alignment_12d_1_Enriched.ifc

```diff
@@ -0,0 +1,69 @@
+#10 = IFCSIUNIT(*, .VOLUMEUNIT., $, .CUBIC_METRE.);
+#11 = IFCSIUNIT(*, .PLANEANGLEUNIT., $, .RADIAN.);
+#12 = IFCGEOMETRICREPRESENTATIONCONTEXT($, 'Model', 3, 1.E-6, #14, #18);
+#13 = IFCLOCALPLACEMENT($, #14);
+#14 = IFCAXIS2PLACEMENT3D(#15, #16, #17);
+#15 = IFCCARTESIANPOINT((0., 0., 0.));
+#16 = IFCDIRECTION((0., 0., 1.));
+#17 = IFCDIRECTION((1., 0., 0.));
+#18 = IFCDIRECTION((0., 1., 0.));
+#19 = IFCSITE('2iTRVf0N96fejtvVjbiFZR', #2, 'Default Site', 'Description of Default Site', $, #13, $, $, .ELEMENT., $, $, $, $, $);
+#20 = IFCRELAGGREGATES('1YL2QP0e55ivtEE6H_3Mm4', #2, 'ProjectContainer', 'ProjectContainer for Sites', #1, (#19));
+#21 = IFCFACILITY('09_nT1I4r0nup4KFCG4Dt7', #2, 'SA  no VG', 'Description of SA  no VG', $, #13, $, $, .ELEMENT.);
+#22 = IFCRELAGGREGATES('3Edrkwm61CrO1NEpyuWmSU', #2, 'SiteContainer', 'SiteContainer For All Models', #19, (#21));
+#23 = IFCRELCONTAINEDINSPATIALSTRUCTURE('2VLpKdG951IPpTkWgVrw_A', #2, 'Building Container', 'BuildingContainer for Elements', (#24), #21);
+#24 = IFCALIGNMENT('3TsZtwmXbER8Q1PJcHGYg2', $, 'RS01 no VG', '12d Alignment String', 'clothoid', $, #59, $, $);
+#25 = IFCRELAGGREGATES('17txXWW0189Rb48wJLlvZl', #2, '', '', #24, (#26));
+#26 = IFCALIGNMENTHORIZONTAL('1aYHKEAr9ANuVigr7mUQoM', #2, 'Horizontal Alignment', '12d Horizontal Alignment', $, $, $, 0., (#28, #30, #32));
+#27 = IFCRELNESTS('0S6tA9lzrFWwczpMhigqbD', #2, '', '', #26, (#28, #30, #32));
+#28 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #29, IFCPLANEANGLEMEASURE(0.), 0., 0., IFCPOSITIVELENGTHMEASURE(48.5550856436632), $, .LINE.);
+#29 = IFCCARTESIANPOINT((0., 0.));
+#30 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #31, IFCPLANEANGLEMEASURE(0.), 150., 150., IFCPOSITIVELENGTHMEASURE(99.118453223473), $, .CIRCULARARC.);
+#31 = IFCCARTESIANPOINT((48.5550856436632, 0.));
+#32 = IFCALIGNMENTHORIZONTALSEGMENT($, $, #33, IFCPLANEANGLEMEASURE(6.60789688156487E-1), 0., 0., IFCPOSITIVELENGTHMEASURE(36.0401064393831), $, .LINE.);
+#33 = IFCCARTESIANPOINT((140.616161880479, 31.5738278837011));
+#34 = IFCCOMPOSITECURVE((#35, #43, #51), $);
+#35 = IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE., #36, IFCNONNEGATIVELENGTHMEASURE(0.), IFCNONNEGATIVELENGTHMEASURE(48.5550856436632), #39);
+#36 = IFCAXIS2PLACEMENT2D(#37, #38);
+#37 = IFCCARTESIANPOINT((0., 0.));
+#38 = IFCDIRECTION((1., 0.));
+#39 = IFCLINE(#40, #41);
+#40 = IFCCARTESIANPOINT((0., 0.));
+#41 = IFCVECTOR(#42, 1.);
+#42 = IFCDIRECTION((1., 0.));
+#43 = IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE., #44, IFCNONNEGATIVELENGTHMEASURE(0.), IFCPARAMETERVALUE(6.60789688156487E-1), #47);
+#44 = IFCAXIS2PLACEMENT2D(#45, #46);
+#45 = IFCCARTESIANPOINT((48.5550856436632, 0.));
+#46 = IFCDIRECTION((1., 0.));
+#47 = IFCCIRCLE(#48, 150.);
+#48 = IFCAXIS2PLACEMENT2D(#49, #50);
+#49 = IFCCARTESIANPOINT((0., 150.));
+#50 = IFCDIRECTION((0., -1.));
+#51 = IFCCURVESEGMENT(.CONTSAMEGRADIENTSAMECURVATURE., #52, IFCNONNEGATIVELENGTHMEASURE(0.), IFCNONNEGATIVELENGTHMEASURE(36.0401064393831), #55);
```

Comment:
the last segment should have the transition code = DISCONTINUOUS

Reply:
Correct, my mistake. Will fix this in the geometry generator.